static/templates/stream_types.hbs

```diff
@@ -8,6 +8,12 @@
             {{t '<b>Public:</b> anyone can join; anyone can view complete message history without joining' }}
         </label>
     </li>
+    <li>
+        <label class="radio">
+            <input type="radio" name="privacy" value="is-web-public" {{#if is_web_public}}checked{{/if}}/>
+            {{t '<b>Web public:</b> anyone (including guest users) can join; can view complete message history without joining' }}
+        </label>
```

Comment:
This should say clearly that anyone on the Internet can access complete message history without creating an account.  We may defer merging this commit until we've made that the case.